odh/base/monitoring/github-receiver-service.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-receiver-service
+spec:
+  ports:
+    - port: 9393
+      protocol: TCP
+      targetPort: 9393
```

Comment:
Instead of this, you can refer to the port by its `name` directly (the name, your setting in `containerPorts` in your deployment spec). It's less error prone having the port number and protocol defined only once.